libexec/scoop-alias.ps1

```diff
@@ -44,13 +44,16 @@ function add_alias($name, $command) {
     # get current aliases from config
     $aliases = init_alias_config
     if ($aliases.$name) {
-        abort "Alias $name already exists."
+        abort "Alias '$name' already exists."
     }
 
     $alias_file = "scoop-$name"
 
     # generate script
     $shimdir = shimdir $false
+    if (Test-Path "$shimdir\$alias_file.ps1") {
+        abort "File '$alias_file.ps1' already exists in shims directory."
```

Comment:
Is there a case where this file would exist but `$aliases.$name` wouldn't? (unless of course we manually create it).

Because we're already checking for `$aliases.$name` above.

Reply:
> Is there a case where this file would exist but $aliases.$name wouldn't?

That's why this PR has been created. 

```
> touch ~\scoop\shims\scoop-random.ps1
> scoop help
Command    Summary
-------    -------
alias      Manage scoop aliases
...
random
```

or maybe the alias entries does not need to be stored in the config

Reply:
No but here you are creating this file manually, with a specific naming scheme. 

I don't think users will do that on their own (there's no reason to manually create that file). 

They will always use the `scoop alias <...>` command to do that.

Reply:
When migration.

Copied Scoop dir from another PC without config file.

Reply:
Oh right